genesis.py

```diff
@@ -29,10 +28,10 @@ class Genesis:
     
     def __init__(self):
         
-        read_token()
+        read_config()
         self.wrapper = telegram.Bot(token = self.token)
 
-        db = sqlite3.connect(DB_NAME)
+        db = mysql.connector.connect(user= self.db_user, password= self.db_pass,host= self.db_host,database= self.db_name)
         cursor = db.cursor()
 
         query = ('CREATE TABLE IF NOT EXISTS users '
```

Comment:
Now that we are using MySQL we should use TINYINT for all _track fields and platform